qa-core/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@budibase/qa-core",
+  "email": "hi@budibase.com",
+  "version": "0.0.1",
+  "main": "index.js",
+  "description": "Budibase Integration Test Suite",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Budibase/budibase.git"
+  },
+  "scripts": {
+    "test": "jest --runInBand",
+    "test:watch": "jest --watch",
+    "test:debug": "DEBUG=1 jest",
+    "api:server:setup": "ts-node ../packages/builder/cypress/ts/setup.ts",
+    "api:server:setup:ci": "node ../packages/builder/cypress/setup.js",
+    "api:test:ci": "start-server-and-test api:server:setup:ci http://localhost:4100/builder test",
+    "api:test": "start-server-and-test api:server:setup http://localhost:4100/builder test"
+  },
+  "jest": {
+    "preset": "ts-jest",
+    "testEnvironment": "node",
+    "moduleNameMapper": {
+      "@budibase/backend-core/(.*)": "<rootDir>/../packages/backend-core/$1",
+      "@budibase/backend-core": "<rootDir>/../packages/backend-core/src",
+      "@budibase/types": "<rootDir>/../packages/types/src"
+    },
+    "setupFiles": [
+      "./scripts/jestSetup.js"
+    ],
+    "setupFilesAfterEnv": [
+      "./src/jest.extends.ts"
+    ]
+  },
+  "devDependencies": {
+    "@budibase/types": "^1.3.4",
+    "@types/jest": "^29.0.0",
+    "@types/node-fetch": "^2.6.2",
+    "chance": "^1.1.8",
+    "jest": "^28.0.2",
+    "prettier": "^2.7.1",
+    "start-server-and-test": "^1.14.0",
+    "timekeeper": "^2.2.0",
+    "ts-jest": "28.0.8",
+    "ts-node": "^10.9.1",
+    "tsconfig-paths": "^4.1.0",
+    "typescript": "^4.8.2"
```

Comment:
Pinning all of these dependencies would be good so we don't get drift between the versions used here and those used inside the rest of the backend packages. 

It might also be good to sync the TS version to `4.7.3`, looking at the search result below it's the most commonly used version across all our packages 
![Screenshot 2022-09-06 at 09 00 40](https://user-images.githubusercontent.com/8755148/188579905-c2d05f1a-fde9-42bb-b6f3-bef5696bec87.png)